test/data_to_bytearray_converter_test.dart

```diff
@@ -5,11 +5,13 @@ import 'package:badgemagic/bademagic_module/models/speed.dart';
 import 'package:badgemagic/bademagic_module/utils/byte_array_utils.dart';
 import 'package:badgemagic/bademagic_module/utils/converters.dart';
 import 'package:badgemagic/bademagic_module/utils/data_to_bytearray_converter.dart';
+import 'package:badgemagic/providers/getitlocator.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  DataToByteArrayConverter converter = DataToByteArrayConverter();
+  Future.delayed(const Duration(seconds: 2));
```

Comment:
Is this needed?

Reply:
removed it was testing something and left it by mistake